pandas/core/reshape/reshape.py

```diff
@@ -50,21 +50,21 @@ class _Unstacker(object):
     ...                                    ('two', 'a'), ('two', 'b')])
     >>> s = pd.Series(np.arange(1.0, 5.0), index=index)
     >>> s
-    one  a   1
-         b   2
-    two  a   3
-         b   4
+    one  a    1.0
+         b    2.0
+    two  a    3.0
+         b    4.0
     dtype: float64
 
     >>> s.unstack(level=-1)
-         a   b
-    one  1  2
-    two  3  4
+           a    b
+    one  1.0  2.0
+    two  3.0  4.0
 
     >>> s.unstack(level=0)
        one  two
-    a  1   2
-    b  3   4
+    a  1.0  3.0
```

Comment:
No, no, now it is correct! So your change is perfect. I just noticed that by running the doctests, we actually corrected some errors in the docs, which is its purpose, so that is good :-)